device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfputil.py

```diff
@@ -65,13 +66,51 @@ def get_presence(self, port_num):
         return False
 
     def get_low_power_mode(self, port_num):
+        # Check for invalid port_num
+        if port_num < self.port_start or port_num > self.port_end:
+            return False
 
-        raise NotImplementedError
+        lpm_cmd = "docker exec syncd python /usr/share/sonic/platform/plugins/sfplpmget.py {}".format(port_num)
+
+        try:
+            output = subprocess.check_output(lpm_cmd, shell=True)
+            if 'LMP ON' in output:
```

Comment:
`LMP ON` should be `LPM ON` (see my review comment on sfplpmget.py above).

Reply:
The typo is fixed.